### DISCUSSION TITLE
How to email the github  for copilot eduation program. My application is rejected for incorrect reason.

### DISCUSSION THREAD

**Discussion by jarroncho** [community/community#75193] [GitHub Education]
### Hi everyone,

Hi everyone,
                   I am Taiwanese. My application for copilot teacher program is rejected. Here is the reply from github. 
Hey 卓經綸,

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Nov 11, 2023 for [@jarroncho](https://github.com/jarroncho).

Please correct your [GitHub billing information](https://github.com/settings/billing/payment_information) with your last name exactly as it appears in your academic affiliation document. If you have only a single legal name, enter it in both the first and last name fields.

Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

Please correct your [GitHub billing information](https://github.com/settings/billing/payment_information) with your first name exactly as it appears in your academic affiliation document. Do not use a variation of your name or a nickname.

Your image contains keywords indicating that it may be an acceptance letter. Only proof of current academic enrollment is acceptable.

The response is wrong. But I think it is processed using ai or something. I don't how to correct it. Please help it, if any body has simular situation.

**Comment by yuvalrovnick** [comment]
I have the exact same problem. If you manage to get over this could you please tell me how? Thanks.
